### PR TITLE
Clarify label on Remove liquidity

### PR DESCRIPTION
**Pull request by EndymionJkb**
Changes proposed in this pull request:
- While explaining this modal, realized the 2nd column label is confusing/uninformative
- It's showing their balance - and how much will be withdrawn for the given # of pool tokens

### CODE REVIEW

File: src/locales/en-US.json [hunk 1]
```diff
@@ -239,6 +239,7 @@
     "weightsPct": "Weights percent",
     "willBurn": "This operation will burn",
     "withdraw": "Withdraw",
+    "withdrawalAmt": "Withdrawal amt",
```
Comment (bonustrack) [inline src/locales/en-US.json:242]: We can add a "." here for the abbreviation 
```suggestion
    "withdrawalAmt": "Withdrawal amt.",
```
We could also remove the amount and just show "Withdraw", i think it's still comprehensible and save some space.

Comment (EndymionJkb) [reply src/locales/en-US.json:242]: How about "My Balance" and "Withdrawal Amt"? Then it's balanced and grammatical

Comment (EndymionJkb) [reply src/locales/en-US.json:242]: Could also just be "Withdrawal" I guess

**Comment by EndymionJkb** [comment]
Actually that's fine for the abbreviation. Should probably leave "pool" because myPoolBalance is used elsewhere, and then it's not confusable with the wallet balance

**Review by bonustrack — APPROVED** [review]

**Pull request merged**